src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+function pushMessage(type) {
+  const message = document.createElement('div');
+
+  message.classList.add('message');
+
+  if (type === 'error') {
+    message.classList.add('error-message');
+    message.textContent = 'Promise was rejected!';
+  } else {
+    message.textContent = 'Promise was resolved!';
+  }
+
+  document.body.appendChild(message);
+}
+
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(Error);
+  }, 3000);
+});
+
+promise1.then(() => pushMessage()).catch(() => pushMessage('error'));
```

Comment:
The `pushMessage` function is called without an argument in the `then` method. This will cause the message to always display 'Promise was resolved!' even if the promise is rejected. Consider passing a type argument to differentiate between resolved and rejected states.